plugin/src/main/java/org/opensearch/ml/cluster/MLSyncUpCron.java

```diff
@@ -97,6 +99,9 @@ public void run() {
         // gather running model/tasks on nodes
         client.execute(MLSyncUpAction.INSTANCE, gatherInfoRequest, ActionListener.wrap(r -> {
             List<MLSyncUpNodeResponse> responses = r.getNodes();
+            if (r.failures() != null && r.failures().size() != 0) {
+                log.error("Received {} failures in the sync up response on nodes", r.failures().size());
```

Comment:
I think you misunderstood me. I mean we should log the error message in r.failures

Reply:
Added the messages in the logging.